scripts/process/json_wu_data.R

```diff
@@ -0,0 +1,7 @@
+process.json_wu_data <- function(viz) {
+  deps <- readDepends(viz)
+  wu_data_15_simple <- deps[["wu_data_15_simple"]]
+  
+  # write to file
+  write(rjson::toJSON(wu_data_15_simple), file=viz[["location"]])
```

Comment:
I think I was picturing more of a json format where each county had it's own object. More like:
```
state_names <- unique(wu_data_15_simple$STATE)
wu_list_for_json <- lapply(state_names, function(s) {
    state_df <- dplyr::filter(wu_data_15_simple, STATE == s)
    county_names <- unique(state_df$COUNTY)
    state_list <- lapply(county_names, function(c) {
        county_df <- dplyr::filter(state_df, COUNTY == c)
        return(county_df)
    })
    names(state_list) <- county_names
    return(state_list)
})
names(wu_list_for_json) <- state_names
write(jsonlite::toJSON(wu_list_for_json, pretty=TRUE), file=viz[["location"]])
```
Which looks like:
```
{
  "AL": {
    "Autauga County": [
      {
        "STATE": "AL",
        "STATEFIPS": "01",
        "COUNTY": "Autauga County",
        "COUNTYFIPS": "001",
        "FIPS": "01001",
        "YEAR": "2015",
        "countypop": 55.347,
        "total": 56.41,
        "thermoelectric": 11.86,
        "publicsupply": 3.64,
        "irrigation": 3.52,
        "industrial": 34.8
      }
    ],
    "Baldwin County": [
      {
        "STATE": "AL",
        "STATEFIPS": "01", 
...
```
@aappling-usgs curious how you were picturing it.